data-structure-implementations/binarysearchtree/bst.go

```diff
@@ -89,6 +100,8 @@ func (bst *BinarySearchTree) LowestCommonAnchestor(item1 *Item, item2 *Item) *It
 func  findLCA(cur *Item, item1 *Item, item2 *Item) *Item {
```

Comment:
If any of the item does not exist within the BST. that case is not handled correctly.